src/components/Country/CountryPage.vue

```diff
@@ -23,7 +23,7 @@
 
 		<div class="row items-center">
 			<q-avatar rounded style="width:auto" :size="isWebApp?'lg':'md'">
-				<img  :src="('countryflagsnew/'+countryCodes_rev[countryKey]+'.svg')">
+				<img :src="('countryflags/Flag_of_'+countryKey+'.svg.png').split(' ').join('_')">
```

Comment:
Hmm, wouldn't we be better off not duplicating this logic of how to get a flag filename for a country in so many files? Next time you get a different set of filenames you have another bunch of files to change, and things will break again.

I'd imagine we define a `function flagUrlFor(countryCode)` somewhere in a Javascript file (in the country store perhaps?) and use that same function in our `.vue` files.

Reply:
Or instead of the country store I'm now also thinking of a mixin. I'm not really familiar enough with Vue/Vuex yet to decide which is most suitable.

Reply:
Orrrr you could even go Vue all the way and make it so that you can do:

```
<flag country="nl">
```

Reply:
yes that would indeed be a good solution!